_docker/drupal/drupal-filesystem/web/config/sync/core.entity_form_display.assembly.connectors.default.yml

```diff
@@ -0,0 +1,64 @@
+uuid: 9f61694d-04c5-4f6b-8342-1dbf5128af18
+langcode: en
+status: true
+dependencies:
+  config:
+    - assembly.assembly_type.connectors
+    - field.field.assembly.connectors.field_audience_selection
+  module:
+    - field_layout
+    - layout_discovery
+third_party_settings:
+  field_layout:
+    id: layout_disabled
+    settings: {  }
+id: assembly.connectors.default
+targetEntityType: assembly
+bundle: connectors
+mode: default
+content:
+  field_audience_selection:
+    weight: 100
+    settings: {  }
+    third_party_settings: {  }
+    type: options_select
```

Comment:
This should be checkboxes to match the rest of the site.

Reply:
I'll take care of that. Thanks

Reply:
This just needs to get moved up in order to be directly underneath the `name` field.

Reply:
Good point. Thank you! I just pushed up that change.

Reply:
looks good